templates/registry/markdown/attribute_macros.j2

```diff
@@ -20,8 +20,17 @@
 {% macro name_with_link(attribute, attribute_registry_base_url, lineage_attributes) %}[{{name(attribute)}}]({{attribute_registry_base_url}}/{{ find_lineage(attribute.name, lineage_attributes) | split_id | list | reject("eq", "registry")| first | kebab_case }}.md){% endmacro %}
 
 {% macro examples(attribute) %}{%- if attribute.examples %}
+{%- if "[]" in attribute.type and "template" not in attribute.type %}
 {%- if attribute.examples is sequence %}
-{%- for example in attribute.examples %}{%if loop.first == false %}; {% endif %}`{{ example }}`{%- endfor %}
+{%- set ns = namespace(all_non_sequence=true) %}
+{%- for example in attribute.examples %}{%- if example is sequence %}{%- set ns.all_non_sequence = false %}{%- endif %}{%- endfor %}
```

Comment:
nit: is it possible to simplify it with [`select` or `selectattr`](https://jinja.palletsprojects.com/en/3.1.x/templates/#jinja-filters.select) `{%- for example in attribute.examples  | selectattr(...) %}`

not sure if it's possible to express `is sequence` there though

Reply:
This is minijinja so doesn't have quite so many features as jinja. However, it does have [select and selectattr](https://docs.rs/minijinja/latest/minijinja/filters/index.html). The logic here though needs to determine if any of the entries in the array are an array - not to filter them. This is the only way to distinguish between a single array example and a list of array examples.

I'll see if there's some other way to express this.

Reply:
Found a way to use `select`.